packages/api-client/src/components/ViewLayout/ViewLayoutContent.vue

```diff
@@ -1,6 +1,25 @@
+<style scoped>
+.view-layout-content > :not([hidden]):not(:first-child) {
+  border-top-width: calc(calc(var(--scalar-border-width) / 2));
+  border-left-width: 0px;
+}
+.view-layout-content > :not([hidden]):first-child {
+  border-bottom-width: 0px;
+}
+
+@screen xl {
+  .view-layout-content > :not([hidden]):first-child {
+    border-right-width: 0px;
+  }
+  .view-layout-content > :not([hidden]):not(:first-child) {
+    border-top-width: 0px;
+    border-left-width: calc(calc(var(--scalar-border-width) / 2));
+  }
+}
+</style>
 <template>
   <div
-    class="border-t-1/2 md:border-t-0 divide divide-y xl:divide-y-0 xl:divide-x-1/2 flex xl:flex-row flex-col custom-scroll pr-0">
+    class="view-layout-content border-t-1/2 md:border-t-0 flex xl:flex-row flex-col custom-scroll pr-0">
```

Comment:
I think we can also avoid the scoped styles here and stick to some (slightly cursed) tailwind instead:

```suggestion
    class="*:border-t-1/2 first:*:border-t-0 xl:*:border-t-0 xl:*:border-l-1/2 xl:first:*:border-l-0 flex xl:flex-row flex-col custom-scroll pr-0">
```

Also for this fix we need to remove the `md:` from `md:border-b-1/2` on [RequestSubpageHeader.vue Line 22]( https://github.com/scalar/scalar/blob/main/packages/api-client/src/views/Request/RequestSubpageHeader.vue#L22) (which should be removed anyway to be honest).

Would be good to add a comment above as well:
```html
<!-- Using *:border-* to avoid chrome performance issue with adjacent sibling selector (~) -->
```